platform/android/detect.py

```diff
@@ -76,40 +77,40 @@ def get_flags():
 # Check if Android NDK version is installed
 # If not, install it.
 def install_ndk_if_needed(env: "SConsEnvironment"):
-    print("Checking for Android NDK...")
     sdk_root = env["ANDROID_HOME"]
     if not os.path.exists(get_android_ndk_root(env)):
         extension = ".bat" if os.name == "nt" else ""
         sdkmanager = sdk_root + "/cmdline-tools/latest/bin/sdkmanager" + extension
         if os.path.exists(sdkmanager):
             # Install the Android NDK
-            print("Installing Android NDK...")
+            print_info("Installing Android NDK...")
             ndk_download_args = "ndk;" + get_ndk_version()
             subprocess.check_call([sdkmanager, ndk_download_args])
         else:
-            print("Cannot find " + sdkmanager)
-            print(
-                "Please ensure ANDROID_HOME is correct and cmdline-tools are installed, or install NDK version "
+            print_error(
+                "Cannot find "
+                + sdkmanager
+                + "Please ensure ANDROID_HOME is correct and cmdline-tools are installed, or install NDK version "
                 + get_ndk_version()
                 + " manually."
             )
-            sys.exit()
+            sys.exit(200)
```

Comment:
Why `200` here specifically? We seem to use `255` throughout otherwise.

Reply:
Ah I see we had this:
```
platform/windows/detect.py
305:        sys.exit(200)
```
That's the only occurrence, so I'd tend to say it was a mistake and not an intentional error reporting API.